HISTORY.rst

```diff
@@ -25,6 +25,60 @@ unreleased
   are supported. Once registered, the device will show up in the linked IoT Hub and can be interacted with or
   simulated using other `az iot device` commands.
 
+**IoT Central updates**
+
+* Add support for device groups CRUD.
+  - az iot central device-group
+    - az iot central device-group list
+    - az iot central device-group show
+    - az iot central device-group create
+    - az iot central device-group delete
+    - az iot central device-group update
+
+* Add support for device attestation CRUD.
+  - az iot central device attestation
+    - az iot central device attestation show
+    - az iot central device attestation create
+    - az iot central device attestation delete
+    - az iot central device attestation update
+
+* Add support for device/module properties/telemetry/command.
+  - az iot central device list-components
+  - az iot central device telemetry
+    - az iot central device telemetry show
+  - az iot central device component-telemetry
```

Comment:
we shouldn't create separated command for device telemetry, use ```az iot central device telemetry show --module-name xxx --component-name yyy --name temperature --device-id --app-id```

Reply:
And please also include a new commands ```az iot central device list-modules``` to show module name list.

Reply:
Sure. Will add it.